spec/restaurant_spec.rb

```diff
@@ -0,0 +1,8 @@
+require './lib/models/restaurant'
+
+describe Restaurand do
```

Comment:
Also, this is well outside of the scope of this feature, as I understood it. It's important that you guys identify what should be happening *in each feature* so that you know when you are done working and can submit the code for review.